src/modules/payment/screens/payment-screen.tsx

```diff
@@ -4,6 +4,7 @@ import React, { useRef } from 'react';
 import { Image as RNImage } from 'react-native';
 import { Checkbox } from 'react-native-paper';
 
+import { usePurchaseOrder } from '../../../api/purchase/use-purchase';
```

Comment:
just a suggestion, maybe we could simplify the imports using path alias, leave you a post here -> https://javascript.plainenglish.io/react-native-path-aliasing-with-simplify-your-imports-and-enhance-your-codebase-9897efee96a8

Reply:
I will solve that in the next PR